index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <!-- Header -->
+  <header>
+    <a href="/15-Sprint-Mission/index.html"
```

Comment:
❗️ 수정요청
해당 태그를 클릭해보시면 잘 동작하시는지 모르겠습니다.
다만 이는 root에서 15-Sprint-Mission 으로 이동해 해당 폴더의 index.html로 가라는 의미입니다.
이는 환경에 따라 다르게 동작할 수 있고 배포시 동작하지 않을 확률이 큽니다.
따라서 아래처럼 수정하시는 것을 추천드립니다~

<img width="372" alt="스크린샷 2025-03-11 오후 3 57 54" src="https://github.com/user-attachments/assets/493bb62a-efa5-40ed-8a6b-5547ff7e646d" />

```suggestion
    <a href="/index.html"
```